### PR TITLE
Update Snippet page styles according to new designs

### PR DESCRIPTION
**Pull request by lotrien**

### CODE REVIEW

**Review by AinuraAliieva — APPROVED** [review]

File: src/components/Snippet.jsx [hunk 1]
```diff
@@ -18,6 +18,9 @@ class Snippet extends React.Component {
     this.state = { isShowEmbed: false };
     this.toggleEmbed = this.toggleEmbed.bind(this);
     this.download = this.download.bind(this);
+    this.copyClipboard = (e) => {
+      copyToClipboard(e, 'embedded');
```
Comment (ikalnytskyi) [inline src/components/Snippet.jsx:22]: Why it's here? It must be a separate commit! Please, do not make different changes in a single commit.
